GVFS/GVFS.Platform.Mac/ProjFSKext.cs

```diff
@@ -73,5 +69,73 @@ public bool TryPrepareFolderForCallbacks(string folderPath, out string error, ou
 
             return true;
         }
+
+        private ProcessResult Bash(string cmd)
+        {
+            string escapedArgs = cmd.Replace("\"", "\\\"");
+
+            ProcessStartInfo startInfo = new ProcessStartInfo
+            {
+                FileName = "/bin/bash",
+                Arguments = $"-c \"{escapedArgs}\"",
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+                UseShellExecute = false,
+                CreateNoWindow = true,
+            };
+
+            using (Process process = new Process())
+            {
+                process.StartInfo = startInfo;
+                string errors = string.Empty;
+                process.ErrorDataReceived += (sender, args) =>
+                {
+                    if (args.Data != null)
+                    {
+                        errors += args.Data + "\r\n";
+                    }
+                };
+
+                process.Start();
+                process.BeginErrorReadLine();
+                string output = process.StandardOutput.ReadToEnd();
+                process.WaitForExit();
+                return new ProcessResult(output, errors, process.ExitCode);
+            }
+        }
+
+        private bool TryLoadKext(ITracer tracer, string enlistmentRoot, out string error)
+        {
+            Console.WriteLine("Kernel extension not loaded.  Attempting to load...");

```

Comment:
Added this message because the `sudo` will prompt for a password and I wanted some kind of message for the user to know why they were being prompted.

Reply:
Is it worth adding "You may getting prompted as this calls sudo" or similar?

Reply:
I'm not sure we can rely on `Console.WriteLine` to do the right thing at this level. Should we send a failure back up to the MountVerb and allow the verb to interact with the user? My concern isn't with this PR -- it's when, in the future, we bring automount online.  We'd still `sudo` but the user would have no clue why they got the prompt. As part of enabling automount on mac work, we should teach GVFS.Service.UI to pop mac notifications, and then automount can go that route and mounts initiated from the command line will continue to interact on the terminal. Thoughts?

Reply:
Yeah this was kind of the quick and dirty way to get it up and working.  I will look into launching kextutil without bash and getting the sudo prompt so that it would work when we have automount.  Thanks!